api/handler/jwt.go

```diff
@@ -0,0 +1,38 @@
+package handler
+
+import (
+	"log/slog"
+	"time"
+
+	"github.com/gin-gonic/gin"
+	"github.com/golang-jwt/jwt/v5"
+	"opencsg.com/csghub-server/api/httpbase"
+	"opencsg.com/csghub-server/common/config"
+)
+
+func NewJWTHandler(config *config.Config) (*JWTHandler, error) {
+	return &JWTHandler{
+		SigningKey: []byte(config.JWT.SigningKey),
+	}, nil
+}
+
+type JWTHandler struct {
+	SigningKey []byte
+}
+
+func (h *JWTHandler) Create(ctx *gin.Context) {
+	claims := jwt.RegisteredClaims{
```

Comment:
should include current user name in claims

Reply:
Thanks for your review. I have added the `current_user` and `organizations`  to the claims.